seshdash/views.py

```diff
@@ -405,3 +407,13 @@ def get_high_chart_data(user,site_id,sites):
      context_high_data['high_pv_production']= high_pv_production
      print (context_high_data['high_pv_production'])
      return context_high_data
+
+def display_alerts(site_id):
+     alerts = Sesh_Alert.objects.filter(site=site_id, isSilence=False)[:5]
```

Comment:
please use order_by here to make sure they ordered in the order mos recent to oldest.

class QuerySet(model=None, query=None, using=None)[source]¶
Usually when you’ll interact with a QuerySet you’ll use it by chaining filters. To make this work, most QuerySet methods return new querysets. These methods are covered in detail later in this section.

The QuerySet class has two public attributes you can use for introspection:

ordered¶
True if the QuerySet is ordered — i.e. has an order_by() clause or a default ordering on the model. False otherwise.